snappy/install.go

```diff
@@ -45,13 +45,34 @@ const (
 	AllowGadget
 )
 
-func doUpdate(part Part, flags InstallFlags, meter progress.Meter) error {
-	if _, err := part.Install(meter, flags); err == ErrSideLoaded {
+func installRemote(mStore *SnapUbuntuStoreRepository, remoteSnap *RemoteSnapPart, flags InstallFlags, meter progress.Meter) (string, error) {
+	downloadedSnap, err := mStore.Download(remoteSnap, meter)
+	if err != nil {
+		return "", fmt.Errorf("can not download %s: %s", remoteSnap.Name(), err)
```

Comment:
Trivial nitpick: `can not download %s` => `cannot download %q`, here and below?

This is silly, but if we manage the standardize the whole code base around error terminology, it ends up feeling a bit more polished.